src/components/tracked-controls.js

```diff
@@ -43,6 +44,7 @@ module.exports.Component = registerComponent('tracked-controls', {
     var controllerEuler = new THREE.Euler();
     var controllerPosition = new THREE.Vector3();
     var controllerQuaternion = new THREE.Quaternion();
+    var deltaControllerPosition = new THREE.Vector3();
```

Comment:
This has been always kind of crappy. Can we create a closure and initialize and reuse these variables instead instancing once per frame? Like this:

https://github.com/aframevr/aframe/blob/master/src/components/raycaster.js#L142